.github/workflows/randomized_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Randomized tests
+on:
+  schedule:
+    - cron:  '42 3 * * *'
+  workflow_dispatch:
+jobs:
+  randomized_tests:
+    name: Randomized tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: |
+          pip install -U -r requirements.txt -c constraints.txt
+          pip install -U -r requirements-dev.txt coveralls -c constraints.txt
+          pip install -c constraints.txt -e .
+          pip install "qiskit-ibmq-provider" -c constraints.txt
+          pip install "qiskit-aer"
+      - name: Prepare executables
+        run: |
+          pip install -U pip
+          python setup.py build_ext --inplace
+      - name: Run randomized tests
+        run: make test_randomized
+      - name: Create comment on failed test run
+        if: ${{ failure() }}
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: 2645
+          body: |
+            Randomized tests check failed:
```

Comment:
Can we put the short commit of the checkout in the body, so there's no ambiguity, and so we still have that information after the run logs are removed?